### PR TITLE
Adding -wait funtionality for issue #21

### PR DESCRIPTION
**Pull request by Lunsford94**
Changes to allow clientside '-wait' flag to wait until sender is present.

### CODE REVIEW

File: connect.go [hunk 1]
```diff
@@ -201,6 +208,10 @@ func (c *Connection) runClient() error {
 				message = receiveMessage(connection)
 				m := strings.Split(message, "-")
 				encryptedData, salt, iv, sendersAddress := m[0], m[1], m[2], m[3]
+                                if sendersAddress == "0.0.0.0" {
+                                        notPresent = true
```
Comment (schollz) [inline connect.go]: add a time.Sleep(1 * time.Second) here to help thwart attackers who want to spam for connections

Comment (Lunsford94) [reply connect.go]: Added.

**Comment by schollz** [comment]
Also it looks like your entire files are in conflict? I think that you just need to run `go fmt *.go` and that should fix it.

**Comment by Lunsford94** [comment]
I noticed. Older version of 'go fmt' apparently; but reformatted with newer install.

**Comment by schollz** [comment]
Looks much better :) I will resolve the conflicts and merge

**Pull request merged**